src/sentry/static/sentry/app/components/pill.tsx

```diff
@@ -127,4 +127,23 @@ const PillValue = styled(PillName)<PillValueProps>`
   }
 `;
 
+const StyledPill = styled('li')<{type?: PILL_TYPE}>`
+  white-space: nowrap;
+  margin: 0 10px 10px 0;
+  display: flex;
+  border: 1px solid ${p => p.theme.borderDark};
+  border-radius: ${p => p.theme.button.borderRadius};
+  box-shadow: ${p => p.theme.dropShadowLightest};
+  line-height: 1.2;
+  max-width: 100%;
+  :last-child {
+    margin-right: 0;
+  }
+
+  ${getPillBorder};
+
+  ${PillValue} {
+    ${getPillValueColor};
```

Comment:
This is missing a `}` - but why do we call `getPillValueColor` here instead of moving into the `PillValue` styled component?

Reply:
oh yeah thanks